spm/quickstart/com.flatpak.quickstart.json

```
@@ -0,0 +1,39 @@
+{
+    "app-id": "com.flatpak.quickstart",
```

```suggestion
    "app-id": "org.flatpak.quickstart",
```

and the file should be renamed.

I know it's just an example, but at least it should conform to the requirements.

---

oh, I should fix that one then.